include/eve/module/core/regular/load.hpp

```diff
@@ -6,95 +6,4 @@
 //==================================================================================================
 #pragma once
 
-#include <eve/arch.hpp>
```

Comment:
Maybe actual load should be in this file instead of detail.

Reply:
It is this way cause of our adage of having the basics of wide, etc, not depending on the module/core. When everything settles on this front, we can revisit that. Also, the doc should be here.